h/annotation_sync.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
You don't seem to like file/module/package names with multiple words :stuck_out_tongue: 
I like `h.indexer` though, renamed it to that :+1: